### PR TITLE
x11-themes/papirus-icon-theme: bumping to version 20210302

### PR DESCRIPTION
**Pull request by ScardracS**
Package-Manager: Portage-3.0.17, Repoman-3.0.2
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+src_install() {
+	insinto /usr/share/icons
+	doins -r ePapirus Papirus{,-Dark,-Light}
+
+}
+
+pkg_preinst() { gnome2_icon_savelist; }
+pkg_postinst() { gnome2_icon_cache_update; }
+pkg_postrm() { xdg_icon_cache_update; }
```
Comment (juippis) [inline x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild]: Just inherit *xdg.eclass* and you can remove all of these phases from this ebuild.

File: x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gnome2-utils xdg-utils
+
+DESCRIPTION="icon theme for Linux"
```
Comment (juippis) [inline x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild]:
```suggestion
DESCRIPTION="Free and open source SVG icon theme"
```

Comment (ScardracS) [reply x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild]: Updated

File: x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild (hunk 1 above)
Comment (ScardracS) [reply x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild]: Thanks, I have also removed gnome2-utils from inherit

Comment (juippis) [reply x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild]: Yes, you should only have `inherit xdg`. That's enough.

**Pull request closed without merging**